.circleci/defaults.sh

```diff
@@ -56,7 +56,7 @@ mapping["chipyard-boom"]=" CONFIG=SmallBoomConfig"
 mapping["chipyard-blkdev"]=" CONFIG=SimBlockDeviceRocketConfig"
 mapping["chipyard-hwacha"]=" CONFIG=HwachaRocketConfig"
 mapping["chipyard-gemmini"]=" CONFIG=GemminiRocketConfig"
-mapping["chipyard-ariane"]=" CONFIG=ArianeConfig"
+mapping["chipyard-cva6"]=" CONFIG=Cva6Config"
```

Comment:
I think every place we have `Ariane` we should have `CVA6` as that seems to be their preferred capitalization.

Reply:
Sure, I can do that.